### DISCUSSION TITLE
Splash screen showing apex icon and channel code name

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Ed/Community#231] [General Discussion]
I am able to see the APEX icon even after replacing all the apex icons from the project with the Dhwani icon.

Scenario:
- when trying to open the app for the first time, able to see dhwani icon
![image-1](https://user-images.githubusercontent.com/107101132/198937907-072c2783-3802-4840-a15c-b4fed757a9ec.png)

- when closing the app and opening again, able to see dhwani icon
![image-2](https://user-images.githubusercontent.com/107101132/198937923-0713407a-094e-42fc-846f-540ded7c830c.png)

- But after opening and closing the app for few times, getting apex icon on splash screen as well as the name is also showing as dhwanifoundation
![image-3](https://user-images.githubusercontent.com/107101132/198937938-31f657d4-f2b0-4bf0-8b25-c5b6f08d757f.png)

cc: @swayangjit @luckynvdu @baljeetsingh @vpPavithra

**Comment by nidhikachhawa24** [comment]
**Scenario:**
When the user has successfully login in the app -> whenever user is opening the app, the spalsh screen is showing apex icon and codename of tenant.

This means that after login, the splash screen is showing the data from some API, how should I change the icon as well as replace the codename with **name** ?

cc: @swayangjit @vpPavithra

**Reply by swayangjit** [reply]
@nidhikachhawa24 The tenant needs to be configured with the required name. The API that is being called is 
`/v1/tenant/info/<tenant_slug>`
